src/artm/core/dictionary.h

```diff
@@ -46,7 +46,7 @@ class Dictionary {
   std::vector<DictionaryEntry> entries_;
   std::unordered_map<Token, int, TokenHasher> token_index_;
   std::unordered_map<int, Token> index_token_;
-  std::unordered_map<int, std::unordered_map<int, int> > cooc_values_;
+  std::unordered_map<int, std::unordered_map<int, std::vector<float> > > cooc_values_;
```

Comment:
What I really like about this solution is that it only adds one line to messages.proto (the line that adds new format to enum). That's all. Also, the user will have to call "ArtmParseCollection" multiple times - once to produce the batches, and then (potentially multiple calls) to produce dictionaries with cooc information. To me this sounds reasonable, because making batches and making dictionaries are independent operations, so I'm very happy to keep this as separate as possible.